src/components/PlayerTag.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+  <div>
+    <v-card min-width="40px" max-width="7vw">
+      <img :src="`https://crafatar.com/avatars/${UUID}?size=300&overlay`"
+      style="width:75%;
```

Comment:
Can we give this element a class and add the stylesheet in a `<style scoped>` section below?